### PR TITLE
Update lucuma-ui and make changes for ViewLike and convert some controls

### PR DESCRIPTION
**Pull request by toddburnside**
In addition to the changes for ViewLike, the PR:

- Changes the inputs in the timing window edit form to use primereact. This required the viewlike changes because `ViewOpt` is involved.
- On the shared target edit warning, switches out the checkboxes for radiobuttons since only one can be selected.

### CODE REVIEW

File: common/src/main/webapp/sass/explore.scss [hunk 1]
```diff
@@ -1258,15 +1287,20 @@ table.explore-border-table {
...
-      .field {
+      .pl-form-field {
         margin-right: 0.5em;
+        width: 9ch;
 
-        .ui.input {
-          width: 9ch;
-        }
+        // .ui.input {
+        //   width: 9ch;
```
Comment (cquiroz) [inline common/src/main/webapp/sass/explore.scss:1295]: Commented code

File: explore/src/main/scala/explore/constraints/TimingWindowsPanel.scala [hunk 2]
```diff
@@ -60,6 +60,8 @@ import java.time.Duration
 import java.time.ZonedDateTime
 import scala.concurrent.duration.FiniteDuration
 
+import scalajs.js.timers
```
Comment (cquiroz) [inline explore/src/main/scala/explore/constraints/TimingWindowsPanel.scala:63]: Is this being used?

**Review by cquiroz — APPROVED** [review]
LGTM thanks

File: explore/src/main/scala/explore/constraints/TimingWindowsPanel.scala (hunk 2 above)
Comment (toddburnside) [reply explore/src/main/scala/explore/constraints/TimingWindowsPanel.scala:63]: It is being used when the code in the `n times` textbox is selected.

**Review by rpiaggio — APPROVED** [review]
LGTM

**Pull request merged**